tutorials/resources/utils/keras_tutorial_tools.py

```diff
@@ -1,164 +0,0 @@
-# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
are you sure that none of the feature tutorials use the utils? (keras or pytorch)